apps/files_versions/src/utils/versions.ts

```diff
@@ -91,7 +91,7 @@ function formatVersion(version: any, fileInfo: any): Version {
 
 	return {
 		fileId: fileInfo.id,
-		label: version.props['version-label'],
+		label: version.props['version-label'].toString(),
```

Comment:
I think using the string constructor is a bit better understandable and prevents `TypeError` on some cases.

```suggestion
		label: String(version.props['version-label']),
```

Reply:
fixed!, thanks :)